src/Microsoft.Azure.WebJobs.Host/Retry/RetryResult.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Hosting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Represents retry settings.
+    /// </summary>
+    public class RetryResult : IOptionsFormatter
```

Comment:
If this is an Options class, I think it should be named with an "Options" suffix like all our other classes

Reply:
Fixed.